packages/language-server/test/plugins/css/CSSPlugin.test.ts

```diff
@@ -324,6 +324,22 @@ describe('CSS Plugin', () => {
                         },
                         newText: 'hsl(240, -101%, 12750%)'
                     }
+                },
+                {
+                    label: 'hwb(240 0% -25400%)',
+                    textEdit: {
+                        range: {
+                            end: {
+                                character: 21,
+                                line: 0
+                            },
+                            start: {
+                                character: 17,
+                                line: 0
+                            }
+                        },
+                        newText: 'hwb(240 0% -25400%)'
+                    }
                 }
```

Comment:
This updated test case comes from the draft PR #1534. I simply adapted it here to make the tests succeed on my local installation. Please do let me know if this is an issue.

Reply:
That test was needed due to an update of another library in the referenced PR, you need to remove it from this PR else the tests fail.

Reply:
Ah, I see. I have since deleted that commit. 👍